app/actions/userActions.js

```diff
@@ -38,35 +76,71 @@ export function deleteUserSuccess(userId) {
 
 export function loadUsers() {
   return dispatch => {
+    dispatch(hideAlertSuccess());
     return userService.loadUsers()
-      .then(data => dispatch(loadUserSuccess(data)));
+      .then(data => dispatch(loadUserSuccess(data)))
+      .catch(error => dispatch(showAlertSuccess(error.description, 'error')));
```

Comment:
suggestion: Maybe We could use a name more generic because we're using it for success and error messages.